src/main/api/2/studio-api-2.yaml

```diff
@@ -1088,6 +1088,55 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /aws/mediaconvert/upload:
+    post:
+      tags:
+        - aws
+      summary: Upload a file to an S3 bucket and trigger a MediaConvert job
+      operationId: uploadVideo
+      description: 'Required Permission: "S3 Write"'
+      requestBody:
+        required: true
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                siteId:
+                  type: string
+                  description: The site ID
+                inputProfileId:
+                  type: string
+                  description: The MediaConvert profile ID
+                outputProfileId:
+                  type: string
+                  description: The S3 profile ID for the generated URLs
+                file:
+                  type: string
+                  format: binary
+                  description: The content of the file to upload
+              required:
+                - siteId
+                - inputProfileId
+                - outputProfileId
+                - file
+      responses:
```

Comment:
No 404? What if the job profiles are not present?

Reply:
Any error from AWS related services will be mapped to a 500 response